net-fs/openafs/openafs-1.8.13.ebuild

```diff
@@ -0,0 +1,348 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
EAPI 8 please.

Reply:
I'm having a build problem when I converted to EAPI 8.  I'm still investigating the reason for the failure (yes the openafs build process is very fragile).  The problem is some unresolved symbols in a link step, but I'm not seeing the reason why for it (yet).  Comparing the runs of an ebuild with EAPI 7 and EAPI 8 isn't showing anything obvious (all the steps are identical up to the point of failure).